### DISCUSSION TITLE
Blackball / Snooker

### DISCUSSION THREAD

**Discussion by BuzzerBlitz** [FairlySadPanda/vrcbce#147] [Q&A] [answered]
Hey will there be any chance of a Blackball table (English Pool) and/or Snooker at any time in the future?
Or would that be something someone else would have to make themselves?

I'm down to give it an attempt although my knowledge is sparse at best.

**Comment by Metamaniac** [comment, marked as answer]
As per https://github.com/VRCBilliards/vrcbce/projects, this is something @FairlySadPanda's wanted for about a year now! (I just edited the default cards out of the project and accidentally made it hard to see the original date 😖 so just take my word for it). There's been a lot of work on the backend to abstract away a lot of the hardcoded game logic (spoilers: **everything** was hardcoded at the beginning) in order to make additional modes like this possible. It's just a matter of time before that becomes a serious possibility; I don't think it's totally out of the question to happen sometime after 1.6 work finishes!
